packages/tracing/src/tracing.ts

```diff
@@ -128,50 +115,45 @@ export async function nodeSDKBuilder(config: TracingConfig) {
 
   const _resource = Resource.default().merge(
     new Resource({
-      [SEMRESATTRS_SERVICE_NAME]: config.serviceName,
-      [SEMRESATTRS_SERVICE_VERSION]: config.release,
+      [ATTR_SERVICE_NAME]: config.serviceName,
+      [ATTR_SERVICE_VERSION]: config.release,
     }),
   );
 
   const _traceExporter = new OTLPTraceExporter({
     //collector url
     url: `http://${config.host}:${config.grpcDefaultPort}`,
   });
-  const _spanProcessors: SpanProcessor[] = [
-    new CustomAWSXraySpanProcessor(
-      _traceExporter,
-      config.samplingRatio ?? 0.01,
-      {
-        // only force 100ms between 2 batch exports.
-        // Default is 5000ms which is 5 seconds and causes us to lose spans
-        scheduledDelayMillis: 100,
-      },
-    ),
-  ];
-  if (config.addSentry) {
-    _spanProcessors.push(new CustomSentrySpanProcessor());
-  }
-  const _idGenerator = new AWSXRayIdGenerator();
+
+  const _metricReader = new PeriodicExportingMetricReader({
+    exporter: new OTLPMetricExporter({
+      url: `http://${config.host}:${config.grpcDefaultPort}`,
+    }),
+    exportIntervalMillis: 10000, // once every 10 seconds, GCP supports 1 every 5 seconds for custom metrics https://cloud.google.com/monitoring/quotas#custom_metrics_quotas
+  });
 
   // set up the default instrumentations for all implementors
   const instrumentations: any[] = [
-    new AwsInstrumentation({
-      suppressInternalInstrumentation: true,
-    }),
-    new DataloaderInstrumentation({}),
-    new ExpressInstrumentation({
-      ignoreLayersType: [ExpressLayerType.MIDDLEWARE],
-    }),
-    new GraphQLInstrumentation({
-      // optional params
-      depth: config.graphQLDepth, //query depth
-      allowValues: true,
-    }),
-    new HttpInstrumentation({
-      ignoreIncomingPaths: ['/.well-known/apollo/server-health'],
+    getNodeAutoInstrumentations({
+      '@opentelemetry/instrumentation-fs': {
+        enabled: false,
+        requireParentSpan: true,
+      },
+      '@opentelemetry/instrumentation-undici': {
+        headersToSpanAttributes: {
+          requestHeaders: [
```

Comment:
These are headers that we add to our trace data, we could expand this in the future. This should be enough to correlate with sentry

Reply:
does applicationname map to consumer key?